invoice.go

```diff
@@ -74,6 +74,18 @@ const (
 	InvoicePaymentSettingsPaymentMethodOptionsCardRequestThreeDSecureAutomatic InvoicePaymentSettingsPaymentMethodOptionsCardRequestThreeDSecure = "automatic"
 )
 
+// The desired country code of the bank account information. Permitted values include: `DE`, `ES`, `FR`, `IE`, or `NL`.
+type InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry string
+
+// List of values that InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry can take
+const (
+	InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountryDE InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry = "DE"
+	InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountryES InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry = "ES"
+	InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountryFR InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry = "FR"
+	InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountryIE InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry = "IE"
+	InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountryNL InvoicePaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry = "NL"
+)
+
```

Comment:
Let's double check with the team - I *believe* we've generally transformed these (similar to currency) to just a regular string instead of an enum, since these names end up being so unwieldly.